### PR TITLE
Reduce injected functions to only split (field) -> (field[254])

### LINKED ISSUE #233: Packing / Unpacking should take/ return array
I think it would more sense to use arrays for `pack128` and `unpack128`in the stdlib.

### PR DESCRIPTION
**Pull request by Schaeff**
Closes #233 , moves 128 bit operations to stdlib

Note: `pack128` is inefficient because linear combinations are not optimized away, they would be once we have #251

### CODE REVIEW

**Comment by stefandeml** [comment]
I had to change the bit space to 253 as:
```log2(21888242871839275222246405745257275088548364400416034343698204186575808495617) = 253.59669135500215
```

**Comment by stefandeml** [comment]
In order to test `pack128` and `unpack128` we need so support for imports in the test framework.

**Comment by stefandeml** [comment]
I've created tests using the new framework as well (using exceptions).
We can add them later after merging https://github.com/Zokrates/ZoKrates/pull/279

File: stdlib/utils/pack/pack128.code [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def main(field[128] bits) -> (field):
+
+	field out = 0
+	field i = 0
+
+	for field j in 0..128 do
+		i = 128 - (j + 1)
```
Comment (Schaeff) [inline stdlib/utils/pack/pack128.code]: I think it would be cleaner for `i` to be declared here

File: stdlib/utils/pack/unpack256.code [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
... b[104], b[105], b[106], b[107], b[108], b[109], b[110], b[111], b[112], b[113], b[114], b[115], b[116], b[117], b[118], b[119], b[120], b[121], b[122], b[123], b[124], b[125], b[126], b[127], b[128], b[129], b[130], b[131], b[132], b[133], b[134], b[135], b[136], b[137], b[138], b[139], b[140], b[141], b[142], b[143], b[144], b[145], b[146], b[147], b[148], b[149], b[150], b[151], b[152], b[153], b[154], b[155], b[156], b[157], b[158], b[159], b[160], b[161], b[162], b[163], b[164], b[165], b[166], b[167], b[168], b[169], b[170], b[171], b[172], b[173], b[174], b[175], b[176], b[177], b[178], b[179], b[180], b[181], b[182], b[183], b[184], b[185], b[186], b[187], b[188], b[189], b[190], b[191], b[192], b[193], b[194], b[195], b[196], b[197], b[198], b[199], b[200], b[201], b[202], b[203], b[204], b[205], b[206], b[207], b[208], b[209], b[210], b[211], b[212], b[213], b[214], b[215], b[216], b[217], b[218], b[219], b[220], b[221], b[222], b[223], b[224], b[225], b[226], b[227], b[228], b[229], b[230], b[231], b[232], b[233], b[234], b[235], b[236], b[237], b[238], b[239], b[240], b[241], b[242], b[243], b[244], b[245], b[246], b[247], b[248], b[249], b[250], b[251], b[252], b[253]]
```
Comment (Schaeff) [inline stdlib/utils/pack/unpack256.code]: fix indentation

File: zokrates_core/src/types/conversions.rs [hunk 3]
```diff
@@ -268,22 +224,12 @@ mod tests {
             assert_eq!(
                 *unpack.statements.last().unwrap(),
                 FlatStatement::Return(FlatExpressionList {
-                    expressions: (FieldPrime::get_required_bits() - nbits
+                    expressions: (FieldPrime::get_required_bits() - 254
```
Comment (Schaeff) [inline zokrates_core/src/types/conversions.rs]: this is zero, change it to that (legacy from generic bit width we don't need anymore)

File: zokrates_core/src/types/conversions.rs [hunk 4]
```diff
@@ -239,22 +197,20 @@ mod tests {
...
+            let unpack: FlatProg<FieldPrime> = split();
             let unpack = &unpack.functions[0];
 
             assert_eq!(unpack.id, String::from("main"));
             assert_eq!(
                 unpack.arguments,
                 vec![FlatParameter::private(FlatVariable::new(0))]
             );
-            assert_eq!(unpack.statements.len(), nbits + 1 + 1 + 1); // 128 bit checks, 1 directive, 1 sum check, 1 return
+            assert_eq!(unpack.statements.len(), 254 + 1 + 1 + 1); // 128 bit checks, 1 directive, 1 sum check, 1 return
```
Comment (Schaeff) [inline zokrates_core/src/types/conversions.rs]: replace with `FieldPrime::get_required_bits()`

**Pull request merged**